static/app/views/replays/detail/tagPanel/useTagFilters.tsx

```diff
@@ -0,0 +1,63 @@
+import {useCallback, useMemo} from 'react';
+
+import {decodeScalar} from 'sentry/utils/queryString';
+import useFiltersInLocationQuery from 'sentry/utils/replays/hooks/useFiltersInLocationQuery';
+import {filterItems} from 'sentry/views/replays/detail/utils';
+import type {ReplayRecord} from 'sentry/views/replays/types';
+
+export type FilterFields = {
+  f_t_search: string;
+};
+
+type Options = {
+  tags: ReplayRecord['tags'];
+};
+
+type Return = {
+  items: ReplayRecord['tags'];
+  searchTerm: string;
+  setSearchTerm: (searchTerm: string) => void;
+};
+
+const FILTERS = {
+  searchTerm: (item: ReplayRecord['tags'], searchTerm: string) =>
+    Object.entries(item).some(str =>
+      JSON.stringify(str).toLowerCase().includes(searchTerm)
```

Comment:
We may want some error handling here

Reply:
JSON.stringify() is only called on the tags, so is error handling still necessary?